### PR TITLE
Javascript objects version

### PR DESCRIPTION
**Pull request by ReemMohamedAbdelfatah**
Hello this is @ReemMohamedAbdelfatah  and @yuvenalmash,
We added these Functionalities:
- Add new Book
- Remove Book
- Show Book
From Local storage.

### CODE REVIEW

**Review by jerryowusu — COMMENTED** [review]
Hi Team, :wave:  

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

invalid


## Invalid Code Review Request

You have submitted a project with one of the `linters (stylelint) failing` and your readme is also missing the `Future Feature section`.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: js/index.js [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+  const newTitle = document.getElementById('addTitleInput').value;
+  const newAuthor = document.getElementById('addAuthorInput').value;
+  const newBook = new Book(newTitle, newAuthor);
+
+  let books = JSON.parse(localStorage.getItem('allEntries'));
+  if (books == null) books = [];
+  localStorage.setItem('entry', JSON.stringify(newBook));
+  books.push(newBook);
+  localStorage.setItem('allEntries', JSON.stringify(books));
+  listBooks();
```
Comment (brytebee) [inline js/index.js:46]: - [x] **Optional:** You could clear the input box after a book is added.

**Review by brytebee — APPROVED** [review]
Hi, @ReemMohamedAbdelfatah & @yuvenalmash👋,

##

### Highlights💡
- Good commit message.✔️ 
- Good PR Summary.✔️
- Great UI design.✔️
- App functions as it should.✔️

##

# STATUS: APPROVED 🕺🥳 

Congratulations🍾🥳!! Your project is ready for presentation!  There is nothing else to say other than... it's time to merge it :shipit: 



Good luck! 🎉 

##

###  **Optional Suggestions:**
- N/A

------

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @brytebee in your question so I can receive the notification.
### _**Use @brytebee on slack for faster responses.**_

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: js/index.js (hunk 1 above)
Comment (ReemMohamedAbdelfatah) [reply js/index.js:46]: We will work on it. Thanks

**Pull request merged**